documentation/host/deployment.rst

```diff
@@ -43,9 +43,6 @@ A more nuanced one from our practice is this:
 
    import os
    project_home = u'/path/to/your/code/flexmeasures'
-   # use this if you want to load your own ``.env`` file.
```

Comment:
We could leave this in, as a suggestion for people - it is indeed not a requirement (anymore). But the comment already phrases that in that way. We could add the word "Suggestion: " in front.